src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -628,6 +629,169 @@ JSON example:
 }
 ```
 
+## Arterial Spin Labeling perfusion data
+
+Template:
+
+```Text
+sub-<label>/[ses-<label>/]
+    perf/
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_asl.nii[.gz]
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_asl.json
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_aslcontext.tsv
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_m0scan.nii[.gz]
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_m0scan.json
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_labeling.jpg
+```
+
+The complete ASL time series should be stored as a 4D NIfTI file in the original acquisition order, accompanied by two ancillary 
+files: `*_asl.json` and `*_aslcontext.tsv`. 
+
+### `*_aslcontext.tsv`
+The `*_aslcontext.tsv` lists the volume_type for each volume in a single column, in the `*_asl.nii[.gz]`, corresponding to the volume_types specified in the table below, based 
+on DICOM Tag (0018,9257) `ASL Context`. Note that the volume_types `control` and  `label` within BIDS only serve to specify the magnetization state of the blood and thus the ASL 
+subtraction order.
+
+| **volume_type** | **Definition**                                                                                                                                                                         |
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| control         | The control image is acquired in the exact same way as the label image, except that the magnetization of the blood flowing into the imaging region has not been inverted.              |
+| label           | The label image is acquired in the exact same way as the control image, except that the blood magnetization flowing into the imaging region has been inverted.                         |
+| m0scan          | The M0 image is a calibration image, used to estimate the equilibrium magnetization of blood.                                                                                          |
+| deltam          | The deltaM image is a perfusion-weighted image, obtained by the subtraction of `control` - `label`.                                                                                    |
+| cbf             | The cerebral blood flow (CBF) image is produced by dividing the deltaM by the M0, quantified into `mL/100g/min` (See also [doi:10.1002/mrm.25197](https://doi.org/10.1002/mrm.25197)). |
```

Comment:
Note that if this change is adopted, it will cascade to other cases. Happy to review for consistency.

Reply:
Hi @effigies , initially we used capitals, but we changed it to this, based on the rest of the specification. It appears that 'not using capitals' is the standard within the specification until now. Also, I suppose this leads to less error in coding.

Reply:
I agree with the readability, as long as this part is consistent with the rest of BIDS; up to the BIDS maintainers :)

Reply:
So  @effigies @sappelhoff, what is the conclusion?

Reply:
We actually have a recommendation on that here: https://bids-specification.readthedocs.io/en/latest/02-common-principles.html#tabular-files

> It is RECOMMENDED that the column names in the header of the TSV file are written in snake_case with the first letter in lower case (for example, variable_name, not Variable_name).

I completely forgot about that at first ... but since it's there, we should adhere to it. Lower case as it is now would be fine with me - but let's hear Chris' comment.

Reply:
These are column contents, not names, right? Or do they need to be column names elsewhere?

Reply:
@effigies They are the volume_names used in the aslcontext.tsv.

Reply:
I don't believe there's any precedent for requiring all TSV contents to be lowercase or snake case.

This really doesn't matter, though. Up to you.

Reply:
@effigies In that case, I think I'd keep it as it is.